src/terminal.ts

```diff
@@ -1,34 +1,28 @@
 // Copyright (c) Jupyter Development Team.
 // Distributed under the terms of the Modified BSD License.
 
+import { Shell } from '@jupyterlite/cockle';
 import { JSONPrimitive } from '@lumino/coreutils';
 
-import { proxy, wrap } from 'comlink';
-
 import {
   Server as WebSocketServer,
   Client as WebSocketClient
 } from 'mock-socket';
 
-import { ITerminal, IRemoteWorkerTerminal } from './tokens';
+import { ITerminal } from './tokens';
 
 export class Terminal implements ITerminal {
   /**
    * Construct a new Terminal.
    */
   constructor(readonly options: ITerminal.IOptions) {
-    this._initWorker();
-  }
-
-  private async _initWorker(): Promise<void> {
-    this._worker = new Worker(new URL('./worker.js', import.meta.url), {
-      type: 'module'
+    this._shell = new Shell({
+      mountpoint: '/drive',
+      driveFsBaseUrl: options.baseUrl,
+      wasmBaseUrl:
+        options.baseUrl + 'extensions/@jupyterlite/terminal/static/wasm/',
```

Comment:
This path is currently hardcoded and works OK for default installations. I am not sure if the extension path is configurable and if so, we'll need to obtain this dynamically at runtime.